cmd/package-operator-manager/components/options_test.go

```diff
@@ -67,3 +67,15 @@ func TestProvideOptions(t *testing.T) {
 		},
 	}, opts)
 }
+
+func TestEnvToInt(t *testing.T) {
```

Comment:
I think there should be two more cases
1. an existing env variable successfully converted to int
2. an existing env variable containing a random string that triggers parse error

Reply:
Added the test cases.